db/migrate/20210929150807_remove_currency_from_gemini_connections.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveCurrencyFromGeminiConnections < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :gemini_connections, :default_currency
+    remove_column :gemini_connection_for_channels, :currency
```

Comment:
Would you `currency` remove from the previous migration? That way we don't create a field and then immediately remove it.

Reply:
Done.